tests/functional/firefox-developer.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global casper */
+'use strict';
+
+var config = require('../lib/config');
+var helpers = require('../lib/helpers');
+var path = '/firefox/developer/';
+var url = config.base() + path;
+
+casper.test.begin('Firefox Developer, Elements: ' + url, 6, function suite(test) {
+
+    casper.start(url, function() {
+        test.assertHttpStatus(200);
+        test.assertElementCount('.download-button', 2, '2 download buttons exist');
+        test.assertExists('.screenshot', 'Browser screenshot exists');
+        test.assertExists('.intro-features li', 'Intro features exist');
+        test.assertExists('.feature-overview .features .feature', 'Features overview exists');
+        test.assertExists('#newsletter-form', 'Newsletter form exists');
+    });
+
+    casper.run(function() {
+        test.done();
+        helpers.done();
+    });
+});
+
+casper.test.begin('Firefox Developer, Click download button: ' + url, 1, function suite(test) {
+
+    casper.start(url, function() {
+        this.click('.intro .download-button .os_win .download-link');
```

Comment:
Yep, I think this test is mostly useful to ensure that no JS errors in the page prevent the download from happening, so which platform link we click doesn't really matter - the logic is the same. 

We currently have existing & more extensive download link tests in the mcom repo. It may make sense to keep the more complete tests there, but something we can look at & discuss in more detail later. Doing complete visual tests for the buttons is something that would be good as a stand alone test suite, but tricky to do as it depends on more than just user agent.